packages/driver-interface/src/botRoutes.ts

```diff
@@ -99,19 +59,14 @@ export const init = (bot: Telegraf<TelegrafContext>): void => {
       case 'picked_up':
       case 'delivered':
       case 'delivery_failed': {
-        const { id: telegramId } = msg.update.callback_query.from
-
+        const telegramId = msg.update.callback_query.from.id
         const { e: event, id: instructionGroupId } = callbackPayload
 
-        const instructionGroup = await cache.getAndDeleteInstructionGroup(
-          instructionGroupId
-        )
-
-        return handleBookingEvent(
-          telegramId,
-          instructionGroup.map((ig) => ig.id),
-          event
-        )
+        return cache
+          .getAndDeleteInstructionGroup(instructionGroupId)
+          .then(([{ id }]) => id)
```

Comment:
This will only mark one booking as delivered, even if instructionGroup had many bookings

Reply:
How do I make instructiongroup be more than one booking?

Reply:
Like the code was before, where we sent in the array of IDs and did a promise.all